modules/core/nuxeo-core-storage-sql/nuxeo-core-storage-sql-extensions/src/main/java/org/nuxeo/ecm/core/storage/sql/db/H2Functions.java

```diff
@@ -55,15 +55,34 @@ public static boolean isInTreeString(Connection conn, String id, String baseId)
         return isInTree(conn, id, baseId);
     }
 
+    /**
+     * @deprecated since 11.1, unused
+     */
+    @Deprecated
     public static boolean isInTreeLong(Connection conn, Long id, Long baseId) throws SQLException {
         return isInTree(conn, id, baseId);
     }
 
+    /**
+     * Compatibility signature without {@code disableVersionACL} parameter
+     */
     public static boolean isAccessAllowedString(Connection conn, String id, String principals, String permissions)
             throws SQLException {
         return isAccessAllowed(conn, id, split(principals), split(permissions));
     }
 
+    /**
+     * @since 11.1
+     */
+    public static boolean isAccessAllowed2(Connection conn, String id, String principals, String permissions,
```

Comment:
Since the signature is not the same maybe it could get rid off the 2 suffix ?
if the args are not splittable and end in single object collections the name would still be relevant.
Or is it because of the alias creation you need a distinct name ?
it seems you can define the arguments types` as in java.lang.Integer.parseInt(java.lang.String, int)`: https://www.h2database.com/html/commands.html#create_alias
Not sure the result would be pretty with a `Connection`, 3 `String` and 1 `boolean` though...

Reply:
Given that this is purely internal I prefer having a short and obviously unambiguous alias definition, so I won't change it.